.github/workflows/nightly.yml

```diff
@@ -385,12 +385,101 @@ jobs:
           name: artifact-macos
           retention-days: 2
 
+  macOS-ARM:
```

Comment:
Since it would seem there will only be Arm based macOS runners going forward, I suggest this job becomes simply "macOS", and to rename the old one to "macOS-Intel"?

Reply:
Agreed, jobs renamed.